lib/ramble/docs/configuration_files.rst

```diff
@@ -561,3 +561,21 @@ format is as follows:
 
 Variables can be defined as lists, scalars, or can refer to a variable defined in
 another fully qualified experiment (through the ``cross_ref_var`` syntax).
+
+-------------------
+Experiment Repeats:
+-------------------
+
+The repeats config options within the ``config`` section are used to define a number
+of times each experiment will be repeated. Summary statistics will be calculated for
+the set of repeats. Its format is as follows:
+
+.. code-block:: yaml
+
+    ramble:
+      config:
+        n_repeats: 'int'
+        repeats_success_strict: [True/False]
+
+More information on experiment repeats is found in the :ref:`workspace configuration
```

Comment:
Maybe "More information on experiment repeats is found" --> "More information on using repeats within a workspace can be found"